bookkeeper-server/build.gradle

```diff
@@ -32,16 +32,15 @@ dependencies {
 
     compileOnly depLibs.lombok
     compileOnly depLibs.spotbugsAnnotations
-    implementation depLibs.bcpkixJdk15on
-    implementation depLibs.bcproveExtJdk15on
+    //implementation depLibs.bcpkixJdk15on
```

Comment:
If these are not needed, you can just delete them from the file. No need to comment it out.

Reply:
I have not sent an MR yet. It's still against my own branch. :-) . When I would send PR I will have a clean PR like always.

Reply:
My bad, did not realize this is PR against your own fork repo =)